core/services/headtracker/head_listener.go

```diff
@@ -215,23 +215,6 @@ func (hl *HeadListener) unsubscribeFromHead() error {
 	return nil
 }
 
-// timedUnsubscribe attempts to unsubscribe but aborts abruptly after a time delay
-// unblocking the application. This is an effort to mitigate the occasional
-// indefinite block described here from go-ethereum:
-// https://chainlink/pull/600#issuecomment-426320971
-func timedUnsubscribe(unsubscriber ethereum.Subscription) {
```

Comment:
Could you help me understand why this isn't needed anymore please?

Reply:
yeah it was fixed in geth a long time ago https://github.com/ethereum/go-ethereum/issues/19768